src/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #d2cece;
+  text-align: center;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1rem;
+
+  /* vertical-align:auto; */
+}
+
+#main-container {
+  width: 30%;
+  box-sizing: border-box;
+  border: 1px solid;
+  background-color: #fff;
+  box-shadow: 0 1px white;
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid gray;
+  padding: 5px;
+}
+
+#add-task {
+  padding: 5px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid gray;
+}
+
+input {
+  border: none;
+  width: 95%;
+
+  /* padding: 5px; */
+}
+
+.enter {
+  /* background-color: #000000; */
+}
+
+.task {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 5px;
+  border-bottom: 1px solid gray;
+}
+
+.task > input {
+  width: 5%;
+}
+
+.task > span {
+  width: 90%;
+  text-align: left;
+  padding: 5px;
+}
+
+.option {
+  width: 20px;
+  height: 20px;
+  color: black;
+  background-color: #fff;
+}
+
+.clear-completed {
+  width: 100%;
+  border: none;
+  background-color: transparent;
+  padding: 5px;
```

Comment:
Kindly note that your list should represent a minimalist design replicate that was given, all features don't need to be implemented now but it should at least resemble the framework. Please work on your styling to an extent as it is really different from what is expected 😄 

Your design :
![image](https://github.com/tasnimchy96/todoList.github.io/assets/20537712/e62956b3-eaa4-4c2c-b1f5-c68d99d29050)

Design given :
![image](https://github.com/tasnimchy96/todoList.github.io/assets/20537712/995451e7-41d9-42d9-88ef-f29dafd8e88b)

Reply:
done

Reply:
- [ ] Please make the changes ass requested by the previous reviewer. Your project's UI should resemble that provided by the [wireframe](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Just so you know, the video has more features than the current project. You don't need to style all the features not implemented yet in the video. Currently, you UI still looks like this:
![image](https://github.com/tasnimchy96/todoList.github.io/assets/45460192/a8fa9a9a-b283-4f0b-8521-ae0d10efbad7)